api/blueprints/routes.py

```diff
@@ -0,0 +1,74 @@
+from flask import Blueprint, request, jsonify
+from sqlalchemy import inspect
+from db.models import (
+    Character,
+    Conversation,
+    Genre,
+    Line,
+    Movie,
+)
+
+
+bp = Blueprint('routes', __name__)
+
+
+def object_as_dict(obj):
+    """
+    Convert sqlalchemy row object to python dict.
+
+    Reference: https://stackoverflow.com/a/37350445/8074325
+    """
+    return {c.key: getattr(obj, c.key)
+            for c in inspect(obj).mapper.column_attrs}
+
+
+@bp.route('/moviedb/api/characters/<string:character_id>', methods=['GET'])
```

Comment:
What do you think about storing `/moviedb/api/` as `API_BASE_PATH` in the config?